framework/mesh/cell/cell.cc

```diff
@@ -6,6 +6,8 @@
 #include "framework/runtime.h"
 #include "framework/logging/log.h"
 
+#include <set>
```

Comment:
so there should be an empty line to distinguish between our include files and cpp's, right?

Reply:
I honestly don't have strong feelings either way so long as we are consistent throughout the code. I have a large set of change coming into this commit, so I can make either change.

Reply:
I think single block is probably the best option. I'll update my incoming changes.

Reply:
maybe we can always list the cpp includes first, and then the ones from the code. ok?

Reply:
Sounds good. Easy enough.

Reply:
@ragusa My only issue with this is in class implementation files. Normally we include the class header first. This would require it to be included elsewhere. My preference is most specific to least in the order:

- modules
- framework
- external packages
- standard library

Reply:
We should probably add this to our coding guide. I think the following order is best:

1. Header for this compilation unit (.h file that corresponds to .cc/.cpp file)
2. Other OpenSn headers (as Zach mentioned above, most specific to least)
3. Non-standard, non-system libraries
4. C++ headers
5. C headers

Reply:
This is what I prefer and do as well.